tests/profile.test.js

```diff
@@ -80,7 +80,6 @@ describe('Profile', () => {
         res.should.have.status(status.OK);
         res.body.should.have.property('profile');
         const { profile } = res.body;
-        profile.should.have.property('country', `${country} Democratic Republic of`);
```

Comment:
why did you remove this @murediane ?

Reply:
It was causing the test to fail I had to fix it